lib/foods.js

```diff
@@ -0,0 +1,28 @@
+const $ = require('jQuery')
+const api = 'https://lit-caverns-20261.herokuapp.com'
+
+$(document).on('ready', getFoods());
```

Comment:
there's actually no need to invoke `getFoods` here - you only need to pass the definition of the function as the callback, then when this event fires, it will invoke the callback...

`$(document).on('ready', getFoods)`